src/Blog/View/post.twig

```diff
@@ -16,14 +16,13 @@
 
 		<article class="u-margin-medium"> 
 			<div class="post-page_header clearfix"> 
-					<h1 class="post-description_title">{{ post.title }}</h1>
-					<span class="post-description_date">{{ post.date }}</span> 
+			<h1 class="post-description_title">{{ post.title }}</h1>
```

Comment:
You deleted too much:)  The break should be the same - one tab.
div is nested in article
h1 and span are nested in div

```
<article>
    <div>
        <h1>
        <span>
    </div>
</article>
```

Reply:
ah! I see!